presentation/src/main/java/me/androidbox/presentation/agenda/screen/AgendaScreen.kt

```diff
@@ -113,7 +140,8 @@ fun AgendaScreen(
             yearSelection = true
         ),
         selection = CalendarSelection.Date { localDate ->
-            agendaScreenEvent(AgendaScreenEvent.OnDateChanged(localDate))
+
+            agendaScreenEvent(AgendaScreenEvent.OnDateChanged(localDate.toZoneDateTime()))
```

Comment:
Do you really need this to be a ZonedDate tho? Since you don't really care about the time, just the date

Reply:
@MKiperszmid Would like to keep the zonedDatetime so if they is another user in another time zone that is also using the app. So I would like the app to be time zone aware. @philipplackner

Reply:
Yeah but you'd never care about the time in this case, since this is to just change the date